Samples~/SampleGame/Packages/manifest.json

```diff
@@ -42,5 +42,16 @@
     "com.unity.modules.wind": "1.0.0",
     "com.unity.modules.xr": "1.0.0"
   },
-  "testables": [ "com.amazonaws.gamelift" ]
+  "testables": [
+    "com.amazonaws.gamelift"
+  ],
+  "scopedRegistries": [
+    {
+      "name": "Unity NuGet",
+      "url": "https://unitynuget-registry.azurewebsites.net",
```

Comment:
This appears to be an unofficial unity registry: https://github.com/xoofx/UnityNuGet

Flagging this for our security review

Reply:
It's part of the integrate guide for the Server SDK https://docs.aws.amazon.com/gamelift/latest/developerguide/integration-engines-unity-using.html